sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py

```diff
@@ -49,26 +50,34 @@ def order_results(request_order, responses):
     ordered = [mapping[id] for id in request_order]
     return ordered
 
-def construct_iso8601(start=None, end=None, duration=None):
+def construct_iso8601(timespan=None):
+    if not timespan:
+        return None
+    try:
+        start, end, duration = None, None, None
+        if isinstance(timespan[1], datetime):
+            start, end = timespan[0], timespan[1]
+        elif isinstance(timespan[1], timedelta):
+            start, duration = timespan[0], timespan[1]
+        else:
+            raise ValueError('Tuple must be a start datetime with a timedelta or an end datetime.')
```

Comment:
Could you add some comments to describe the situation? 
e.g. if isinstance(timespan[1], datetime): # if timespan is tuple[datetime, datetime], we treat it as [start time, end time].